drivers/uktty/ns16550/ns16550.c

```diff
@@ -72,6 +73,10 @@
 #define NS16550_LSR_RX_EMPTY	0x01U
 #define NS16550_LSR_TX_EMPTY	0x40U
 
+static const char * const fdt_compatible[] = {
```

Comment:
`*const` no whitespace ? :thinking:

Reply:
Yes, if it was a variable name it wouldn't require space.